src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1691,6 +1684,9 @@ ucs_status_t uct_dc_mlx5_ep_check_fc(uct_dc_mlx5_iface_t *iface,
     status              = uct_dc_mlx5_ep_fc_hard_req_send(ep, fc_entry->seq);
     if (status != UCS_OK) {
         if (ret != UCS_KH_PUT_KEY_PRESENT) {
+            /* Decrement a FC sequence number to roll back changes done above,
+             * because FC_HARD_REQ wasn't sent with this sequence number */
+            iface->tx.fc_seq--;
```

Comment:
is it really important to make sure we never "skip" a sequence number?
or maybe just ++ it after line 1700?

Reply:
> or maybe just ++ it after line 1700?

done